app/scripts/filters/util.js

```diff
@@ -348,6 +348,30 @@ angular.module('openshiftConsole')
           .replace(/^./, function(str){ return str.toUpperCase(); });
     };
   })
+  // event messages should simple, but human readable:
+  // Pulled Pod <name>
+  // Unhealthy Pod <name>
+  // Deployment created for DeploymentConfig <name>
+  // etc.
+  // This filter handles the "Reason <optional preposition>"
+  // but not the <name> as we <a> link that in templates.
+  .filter('messageForEvent', function($filter) {
```

Comment:
> Its definitely better, but does Pulled -- Pod mariadb-4-cbgkr still imply it pulled a pod (instead of an image)?

Yeah, maybe we special case this one. Although I'm debating it because it would be inconsistent with the Events table.